samples/helloworld/gateway-api/README.md

```diff
@@ -0,0 +1,73 @@
+# Configure helloworld using the Kubernetes Gateway API
+
+Istio intends to [replace its traffic management API](https://istio.io/latest/blog/2022/gateway-api-beta/) with the Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/) as the default API for traffic management in the future.
+
+You can use the following instructions to configure the ingress gateway and routing for the helloworld sample.
+
+## Before you begin
+
+The Gateway API CRDs do not come installed by default on most Kubernetes clusters, so install them if not present:
+
+```bash
+kubectl get crd gateways.gateway.networking.k8s.io || \
+  { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.5.0" | kubectl apply -f -; }
+```
+
+Also make sure you are running two versions (v1 and v2) of the helloworld service:
+
+```bash
+kubectl apply -f ../helloworld.yaml
+```
+
+## Configure the helloworld gateway
+
+Apply the helloworld gateway configuration:
```

Comment:
Added a sentence, below.

In general I'm trying not to get into too much detail, since this is just the "hello world" app. I just want people to think, wow, this isn't much different, but now we will have an industry standard API. I envision lots more details in the other docs, starting with Bookinfo and all the traffic management tasks.